src/path.js

```diff
@@ -23,7 +23,10 @@ module.exports = _curry2(function path(paths, obj) {
   } else {
     var val = obj;
     var idx = 0;
-    while (val != null && idx < paths.length) {
+    while (val !== void 0 && idx < paths.length) {
+      if (val === null) {
+        return;
+      }
       val = val[paths[idx]];
       idx += 1;
     }
```

Comment:
It seems we could do this instead:

```diff
@@ -23,8 +23,8 @@ module.exports = _curry2(function path(paths, obj) {
   } else {
     var val = obj;
     var idx = 0;
-    while (val !== void 0 && idx < paths.length) {
-      if (val === null) {
+    while (idx < paths.length) {
+      if (val == null) {
         return;
       }
       val = val[paths[idx]];
```

This would avoid separate code paths for `null` and `undefined`. :)